commonmark/src/main/java/org/commonmark/renderer/text/TextContentWriter.java

```diff
@@ -24,6 +24,12 @@ public void colon() {
         }
     }
 
+    public void pipe() {
```

Comment:
Looks great now! The only thing unsure about now is this method. I'm not sure about adding it to the public API, it seems very arbitrary. Is there a good reason to avoid writing two pipes in a row? Otherwise I'd just do `append('|')` in `renderCell` and remove this method.

Reply:
It will both have the same outcome I guess. I don't know which solution is most in line with the codebase, so you may decide. With the TablesTextContentTest.java testclass you can make sure the output is still the same.